common/autoware_test_utils/test/test_mock_data_parser.cpp

```diff
@@ -581,7 +581,7 @@ TEST(ParseFunction, CompleteFromFilename)
   const auto parser_test_route =
     autoware_test_utils_dir + "/test_data/lanelet_route_parser_test.yaml";
 
-  const auto lanelet_route = parse<LaneletRoute>(parser_test_route);
+  const auto lanelet_route = *parse<std::optional<LaneletRoute>>(parser_test_route);
```

Comment:
```suggestion
  if (const auto lanelet_route_opt = parse<std::optional<LaneletRoute>>(parser_test_route); lanelet_route_opt) {
     const auto lanelet_route = lanelet_route_opt.value();
     // do the tests ...
```

Reply:
Thank you, I changed all the ones in https://github.com/autowarefoundation/autoware.universe/pull/9391/commits/4520801cec4097f9d5361a3fde3942681ffffdb9